### PR TITLE
Provide a powershell script to create a policy in a series of operations

### PR DESCRIPTION
**Pull request by srnavale**
Creates a policy with the default values for policy attributes. Then adds a client, schedule, and backup selection to it. Deletes the client and schedule and finally deletes the policy.

### CODE REVIEW

**Review by akumar91 — COMMENTED** [review]
we should also add section on how to run the script in readme.md file

File: recipes/powershell/New-Policy-StepByStep.ps1 [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+This script can be run using NetBackup 8.1.2 and higher.
+It creates a policy with the default values for policy type specific attributes, adds a client, schedule, and backup selection to it, then deletes the client, schedule and finally deletes the policy.
+.EXAMPLE
+./New-Policy-StepByStep.ps1 -MasterServer "nb-master.example.com" -UserName "administrator" -Password "password" -DomainName "domain name"
+#>
+
+#Requires -Version 4.0
+
+Param (
+    [string]$MasterServer = $(Throw "Please specify the name of the NetBackup Master Server using the -MasterServer parameter."),
```
Comment (akumar91) [inline recipes/powershell/New-Policy-StepByStep.ps1:14]: can we rename  to nbmaster, username, password, domainName and domainType respectively to be consistent with other scripts?

Comment (srnavale) [reply recipes/powershell/New-Policy-StepByStep.ps1:14]: Tried to follow powershell script recommended way of defining command line arguments.

**Review by santhosh-vrts — APPROVED** [review]
Just a general comment (minor) - we can probably keep backupSelection as a global variable (similar to policy/schedule names, and assuming only one backup selection) so it's easier for someone using the script to change it if needed.

**Pull request merged**